static/src/javascripts/es6/projects/common/modules/crosswords/helpers.js

```diff
@@ -1,40 +1,30 @@
-// vendor
 import _ from 'common/utils/_';
 
-// app
 import constants from 'es6/projects/common/modules/crosswords/constants';
 
-function isAcross(clue) {
-    return clue.direction === 'across';
-}
+let isAcross = (clue) => clue.direction === 'across';
```

Comment:
`const`

Though, is there some reason we don't just use native function declarations?